src/collections/_documentation/cli/releases.md

```diff
@@ -3,7 +3,7 @@ title: 'Release Management'
 sidebar_order: 2
 ---
 
-The `sentry-cli` tool can be used for release management on Sentry. It allows you to create, edit and delete releases as well as upload release artifacts for them.
+The `sentry-cli` tool can be used for release management on Sentry. It allows you to create, edit and delete releases as well as upload release artifacts for them. Note that releases are global per organization, so if you want the releases in different projects to be treated as separate entities, you should make the version name unique across the organization.
```

Comment:
Should we add an example? Similar to the one you wrote in the PR description?

Reply:
Should we? 🤔 I was trying to keep it less wordy, is this not clear enough? I'm open to changing it!